joplin/js/CreateContentModal/ChooseTitleStep.js

```diff
@@ -20,7 +20,7 @@ const ChooseTitleStep = ({
       )}
       {pageType !== 'department' && (
         <span>
-          Write an actionable title for your  {pageType} page, starting with a
+          Write an actionable title for your  {pageType.replace("_", " ")} page, starting with a
```

Comment:
this feels like part of some tech debt, but I think it's good for now. 

Ideally in the future it would be great to find a way to pass model data to our modal, since Django already generates the model names and their verbose equivalents IIRC

Reply:
I'd forgotten about verbose_name! That's probably what I'd want here.

Reply:
For sure!
Just for ref: https://docs.djangoproject.com/en/2.2/ref/models/options/
"If this isn’t given, Django will use a munged version of the class name: CamelCase becomes camel case."